port/unix/omrsysinfo.c

```diff
@@ -190,7 +190,12 @@ uintptr_t Get_Number_Of_CPUs();
 #define TICKS_TO_USEC   ((uint64_t)(USECS_PER_SEC/JIFFIES))
 
 /* For the omrsysinfo_env_iterator */
+#if defined(OSX)
+#include <crt_externs.h>
+#define environ (*_NSGetEnviron())
```

Comment:
I think it would be cleaner and clearer to modify `copyEnvToBuffer` to call this rather than playing a `#define` trick here.